### PR TITLE
Look and feel: alignment with other repositories

### PR DESCRIPTION
**Pull request by PFigs**
Update the readme with links to the appropriate documentation.

### CODE REVIEW

File: gateway_to_backend/README.md [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+
+- **Request:**
+
+    > **topic:** gw-request/get_gateway_info/*\<gw-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoReq][message_GetGwInfoReq]
+
+- **Response:**
+
+    > **topics:** gw-response/set_config/*\<gw-id\>/\<sink-id\>*
```
Comment (GwendalRaoul) [inline gateway_to_backend/README.md]: it is not set_config but get_gateway_info.
It was already wrong in pdf version of wp-rm-128...

File: gateway_to_backend/README.md [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+    > **topic:** gw-request/get_configs/*\<gw-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetConfigsReq][message_GetConfigsReq]
+
+- **Response:**
+
+    > **topics:** gw-response/get_configs/*\<gw-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetConfigsResp][message_GetConfigsResp]
+    >              and [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[SinkReadConfig][message_SinkReadConfig]
```
Comment (GwendalRaoul) [inline gateway_to_backend/README.md]: Is the and really needed?

File: gateway_to_backend/README.md [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+
+    > **topic:** gw-request/get_gateway_info/*\<gw-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoReq][message_GetGwInfoReq]
+
+- **Response:**
+
+    > **topics:** gw-response/set_config/*\<gw-id\>/\<sink-id\>*
+    >
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoResp][message_GetGwInfoResp] and [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]
```
Comment (GwendalRaoul) [inline gateway_to_backend/README.md]: Same here for the "and"

File: gateway_to_backend/README.md (hunk 1 above)
Comment (PFigs) [reply gateway_to_backend/README.md]: ok, i'll fix it

File: gateway_to_backend/README.md (hunk 2 above)
Comment (PFigs) [reply gateway_to_backend/README.md]: Good point, a comma should be enough.

File: gateway_to_backend/README.md [hunk 4]
```diff
@@ -265,9 +264,9 @@ to monitor this change._
 
 - **Response:**
 
-    > **topics:** gw-response/set_config/*\<gw-id\>/\<sink-id\>*
+    > **topics:** gw-response/get_gateway_info/*\<gw-id\>/\<sink-id\>*
     >
-    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoResp][message_GetGwInfoResp] and [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]
+    > **content:** [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GetGwInfoResp][message_GetGwInfoResp], [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]
```
Comment (GwendalRaoul) [inline gateway_to_backend/README.md]: My comment was probably unclear :-)

I was suggesting to remove all the second part: [GenericMessage][message_GenericMessage].[WirepasMessage][message_WirepasMessage].[GatewayInfo][message_GatewayInfo]

Point is that if you follow first link [GetGwInfoResp] you will see that one field is GatewayInfo described just above.

If we do it here, why not doing it everywhere alos? Like giving link to what and EventHeader format is for example.

Do you agree? It is just a suggestion ;-)

Comment (PFigs) [reply gateway_to_backend/README.md]: It makes sense to remove it and since we now have the class hierarchy I don't see a need for it anymore.

**Pull request merged**